core/domain/exp_services.py

```diff
@@ -104,17 +104,21 @@ def _migrate_states_schema(versioned_exploration_states):
 
 # Repository GET methods.
 def _get_exploration_memcache_key(exploration_id, version=None):
-    """Returns a memcache key for an exploration."""
+    """
+    Args:
+        - exploration_id: str. The exploration id.
+        - version: int. Version of the exploration , default value None.
+    Returns:
```

Comment:
I guess you mean to say, i should start working on "develop" branch and make the changes which were asked in the previous PR.